packages/account/src/Sections/Security/Passwords/sent-email-modal.jsx

```diff
@@ -0,0 +1,127 @@
+                    { identifier_title }
+                );
+                break;
+            default:
+                subtitle = localize('Please click on the link in the email to reset your password.');
+                break;
+        }
+        return subtitle;
+    };
+
+    return (
+        <Modal
+            className={classNames({ 'sent-email__modal': !is_unlink_modal })}
+            is_open={is_open}
+            has_close_icon={!is_unlink_modal}
+            should_header_stick_body
+            title={
+                is_unlink_modal ? (
+                    <Localize
+                        i18n_default_text='Are you sure you want to unlink from {{identifier_title}}?'
+                        values={{ identifier_title }}
+                    />
+                ) : (
+                    ''
+                )
+            }
+            toggleModal={onClose}
+            width='440px'
+        >
+            {is_unlink_modal ? (
+                <React.Fragment>
+                    <Text className='sent-email__modal-unlink-title' size='xs'>
+                        {localize('You will need to set a password to complete the process.')}
+                    </Text>
+                    <Modal.Footer>
+                        <Button onClick={onClose} has_effect text={localize('Cancel')} secondary large />
+                        <Button has_effect onClick={onClickSendEmail} primary large>
+                            <Localize
+                                i18n_default_text='Unlink from {{identifier_title}}'
+                                values={{ identifier_title }}
+                            />
+                        </Button>
+                    </Modal.Footer>
+                </React.Fragment>
+            ) : (
+                <Div100vhContainer
+                    className='account__scrollbars_container-wrapper'
+                    is_disabled={isDesktop()}
+                    height_offset='80px'
+                >
+                    <Modal.Body>
+                        <div onClick={onClose} className='sent-email__modal-close'>
+                            <Icon icon='IcCross' />
+                        </div>
+                        <SendEmailTemplate
+                            className='sent-email'
+                            title={localize('We’ve sent you an email')}
+                            subtitle={getSubtitle()}
+                            lbl_no_receive={localize("Didn't receive the email?")}
+                            txt_resend={localize('Resend email')}
+                            txt_resend_in={localize('Resend email in {{seconds}}s', { seconds: '{{seconds}}' })}
+                            onClickSendEmail={onClickSendEmail}
+                        >
+                            {getNoEmailContentStrings().map((item, idx) => (
+                                <div className='sent-email__content' key={idx}>
```

Comment:
Avoid using indexes as keys if you can, you can do so by either using the `item.icon` (lazy approach 🦥) or by creating pre-defined keys in the `getNoEmailContentStrings` function (recommended approach 💯). 

Further reading:
- https://kentcdodds.com/blog/understanding-reacts-key-prop/
- https://medium.com/wesionary-team/using-index-as-a-key-is-an-anti-pattern-in-react-8e5db3aea3a9